WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/PostsCardViewModel.swift

```diff
@@ -30,8 +28,6 @@ class PostsCardViewModel: NSObject {
 
     private var status: BasePost.Status = .draft
 
-    private var shouldSync: Bool
-
     private var syncing: (NSNumber?, BasePost.Status)?
 
     private weak var viewController: PostsCardView?
```

Comment:
What do you think about renaming that to simple `view`?

Reply:
Agree! Changed in 9db63f09c797f454eae1df486928e21fb3fb273a